addons/wowl/static/src/components/loading_indicator/loading_indicator.xml

```diff
@@ -1,10 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
 
-    <div t-name="wowl.LoadingIndicator" t-att-class="state.isShown ? 'o_loading' : ''" owl="1">
-        <t t-if="state.isShown">
-            Loading (<t t-esc="state.count"/>)
-        </t>
+    <div t-name="wowl.LoadingIndicator" class="o_loading" t-att-class="{ fadeout: !state.isShown }" owl="1">
```

Comment:
woops, why not a good owl' transition :D ?

Reply:
Good Owl' transitions only helps in one specific situation: when the node is attached/detached from the dom :-)

The loading indicator is always present on the DOM.

Maybe we should change that ? @FrancoisGe , more infos on that please ?

Reply:
@brboi How you want to remove the loading indicator when we don't need it ? If you find a good way to do it, you can do it :)

Reply:
You can check commit V2: this use t-transition by inserting a sub-component. Ugly IMO.

As fge suggested, we could also make the loading indicator a service...

@kebeclibre any thoughts on this ?